server/liveblog/themes/themes.py

```diff
@@ -271,17 +271,24 @@ def publish_related_blogs(self, theme):
         # for t in self.get_children(theme['name']) + [theme['name']]:
         #     terms.append({'term': {'blog_preferences.theme': t}})
         blogs = get_resource_service('blogs').get(req=None, lookup={})
```

Comment:
Actually, you could get blogs which use this theme using the following lookup: {'blog_preferences.theme': theme['name']}